### PR TITLE
CLOUDP-119400 add timestamp in the error messages, handle aes panic

### PR DESCRIPTION
**Pull request by tibulca**
## Proposed changes

_Jira ticket:_ CLOUDP-119400

Closes #[issue number]

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: e2e/cloud_manager/decryption/localKey/test5-output [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{"atype":"decryptionLogError","line":2,"error":"logRecordIdx missmatch: expected 1 got 2","ts":1644232050645}
+{"ts":{"$date":"2022-02-07T11:07:30.645Z"},"atype":"createDatabase","uuid":{"$binary":{"base64":"+mOPbB12RDGgTsXXjR8R/A==","subType":"04"}},"local":{"isSystemUser":true},"remote":{"isSystemUser":true},"users":[],"param":{"ns":"admin"},"roles":[],"result":0}
```
Comment (andreaangiolillo) [inline e2e/cloud_manager/decryption/localKey/test5-output:2]: new line missing

File: e2e/cloud_manager/decryption_localkey_test.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func TestDecrypt(t *testing.T) {
 		req.NoError(err)
 	})
 
-	for i := 1; i <= 4; i++ {
+	for i := 1; i <= 5; i++ {
```
Comment (andreaangiolillo) [inline e2e/cloud_manager/decryption_localkey_test.go:55]: [q] what's the meaning of this number? Can we get it from the code?

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: e2e/cloud_manager/decryption_localkey_test.go (hunk 2 above)
Comment (tibulca) [reply e2e/cloud_manager/decryption_localkey_test.go:55]: this is the number of test cases (see next line `	t.Run(fmt.Sprintf("Test case %v", i)`)
and it corresponds to the number of input files `inputFile := decryption.GenerateFileNameCase(tmp, i, "input")`
Can be replaced with reading the entire folder, filter (regex on file names), and sort, but I'd say it's a bit of overhead for a test and it's also static data.

**Pull request merged**